main.go

```diff
@@ -208,6 +208,23 @@ func main() {
 					goto send
 				}
 
+				// If
+				// MariaDB [(none)]> show all slaves status;
+				// Empty set (0.000 sec)
+				// we assume is a master
+				if r.Resultset.RowNumber() == 0 {
+					log.Info("There is no replication status, looks like master")
```

Comment:
I'm not sure this message requires the `info` level:

```suggestion
					log.Debug("no replication status, looks like master")
```